src/opa.js

```diff
@@ -236,7 +237,7 @@ async function _loadPolicy(policyWasm, memory, customBuiltins) {
     console.error("opa_wasm_abi_minor_version undefined");
   }
 
-  env.instance = wasm.instance ? wasm.instance : wasm;
```

Comment:
Could we not move this line into line 214, and use env.instance in the places we now use `wasmInstance`? 🤔 It would be a little more consistent with below, where we call `env.instance.exports.builtins()`.